### DISCUSSION TITLE
Gift message does not include from who it was sent

### DISCUSSION THREAD

**Discussion by nikolajevs86** [etsy/open-api#1210] [Q&A]
Hi! Recently, we started to receive complaints from the merchants that the gift messages that they create are not full and there is missing information about who sent it.
As we see now Etsy provides an additional input at the checkout page to be able to input the sender, but from the Etsy API v3 order's gift message does not include it.
Is it a known bug or there is some additional functionality on how we can fetch a sender using API?
Thank you!

**Comment by krushr1** [comment]
This is actually unbelievable. 

Etsy is the only shop I've ever seen on the internet that leaves the sender's name anonymous by default.

I'm very proactive about communicating this with customers, but I've seen other sellers receive poor reviews because of this and it's not even their faults.

**Etsy, in your Gift print out, PLEASE include the gift giver's name by default**, and add a checkbox to make it anonymous in the extremely rare case the gift giver would prefer to remain unnamed.

Thank you!
